src/Main/Main.js

```diff
@@ -119,6 +123,7 @@ class Main extends Component {
       initialValue: {
         vendorIDQuery: 'query=(name=null)',
         contactIDs: 'query=(id=null)',
+        interfaceIDs: 'query=(id=null)',
```

Comment:
Basically, it doesn't look like a pattern to repeat. It was implemented for contacts, so we decided to leave if for now untouched. In your case this is a new component, you're able to implement it right, on a component level and to not use legacy code.